ExploreASL_Initialize.m

```diff
@@ -484,9 +484,15 @@
     end
     
     if ~exist(x.D.ROOT, 'dir')
-        warning([x.D.ROOT ' didnt exist as folder, trying path of DataPar file']);
-        x.D.ROOT = pathstr;
-        x.ROOT = pathstr;
+        % Check if x.D.ROOT was defined as a relative path
+        if exist(fullfile(pathstr,x.D.ROOT), 'dir')
+            x.D.ROOT = fullfile(pathstr,x.D.ROOT);
+            x.ROOT = x.D.ROOT;
```

Comment:
Is `x.ROOT` still used? It might be omitted since we're moving all to substructs.

Reply:
I'm not sure, but we still used it in another part a few lines below, so I thought I add it there as well, just to be safe. We can rethink this after the release. I just don't want to break anything with a small fix like this one :)

Reply:
We should keep this in mind for #480.